types/gapi.drive.v3/README.md

```diff
@@ -0,0 +1,16 @@
+# Installation
+> `npm install --save @types/gapi.drive.v3`
+
+# Summary
+This package contains type definitions for Google Drive API v3 (https://developers.google.com/drive/).
+
+# Details
+Files were exported from https://www.github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/gapi.drive.v3
```

Comment:
Directory `types/gapi.drive.v3` should be renamed `types/gapi.drive`.

Also I don't think it's relevant to create a README file for definitions because:
- people don't read it (from my experience)
- when `index.d.ts` is updated, nobody updates the README 

It is better to add comments inside `index.d.ts`
<br>

Here, your README.md brings no value:
- `npm install --save @types/gapi.drive.v3` => already documented at TypeScript/DefinitelyTyped level/StackOverflow...

- `This package contains type definitions for...` => already documented inside `gapi.drive.v3/index.d.ts`

- `Files were exported from https://www.github.com/DefinitelyTyped...` => well...

- `Last updated: Aug, 17 Jun 2017 16:07:48 GMT` => Git already gives this information

- `Dependencies: gapi` => we know that from `index.d.ts`

- `These definitions were written by Tomohiko Ozawa <https://github.com/kota65535>` => already in `index.d.ts`